hawc/templates/base.html

```diff
@@ -105,11 +105,6 @@
     <script type="text/javascript" src="{% static 'vendor/bootstrap/4.5.3/js/bootstrap.min.js' %}"></script>
     <script type="text/javascript" src="{% static 'vendor/jqueryui/1.10.3/jquery-ui.min.js' %}"></script>
     <script type="text/javascript" src="{% static 'vendor/htmx/1.6.1/htmx.min.js' %}"></script>
-    <script>
```

Comment:
moved this method from base.html to the `frontend/shared/startup.js` method; consistent with jQuery ajax